server/fishtest/views.py

```diff
@@ -530,25 +530,25 @@ def users_monthly(request):
 
 
 def get_master_info():
-    message_search = re.compile(r"^.*$", re.MULTILINE)
-    bench_search = re.compile(r"(^|\s)[Bb]ench[ :]+([0-9]+)", re.MULTILINE)
+    bench_search = re.compile(r"(^|\s)[Bb]ench[ :]+([1-9]\d{5,9})(?!\d)")
     for idx, commit in enumerate(
         requests.get(
             "https://api.github.com/repos/official-stockfish/Stockfish/commits"
         ).json()
     ):
         if "commit" not in commit:
             return None
-        bench = bench_search.search(commit["commit"]["message"])
+        message_idx = commit["commit"]["message"].strip().split("\n")
+        bench = bench_search.search(message_idx[-1])
         if idx == 0:
-            message = message_search.search(commit["commit"]["message"])
+            message = "\n".join(line.strip() for line in message_idx)
```

Comment:
My fault, the regex of the original code writes only the first line of the message (but it misses to strip blank spaces)

```diff
- message = "\n".join(line.strip() for line in message_idx)
+ message = message_idx[0].strip()
```